pages/education.py

```diff
@@ -108,6 +112,22 @@ def fetch_edu_sub_func_data_once(edu_data):
     return dash.no_update
 
 
+@callback(
+    Output("stored-data-education-subnational", "data"),
+    Input("stored-data-education-subnational", "data"),
+)
+def fetch_edu_subnational_data_once(edu_data):
+    if edu_data is None:
+
+        # filter shared data down to education specific
+        subnational_data = queries.expenditure_and_outcome_by_country_geo1_func_year()
```

Comment:
Comment says "filter shared data down to education specific" but the code doesn't?